yetanotherco/aligned_layer
# fix(contracts): registrycoordinator pauser is set to eigenlayer addresses

Merge after #481 

# How to test
1. Run anvil
```
make anvil_start_with_block_time
```

2. If you run 
```
cast call --rpc-url http://localhost:8545/ 0xf5059a5D33d5853360D16C683c16e67980206f36 "paused()"
```
the response should be `0x0`

3. Now, set the contract to PAUSE state
```
cast send --rpc-url http://localhost:8545 0xf5059a5D33d5853360D16C683c16e67980206f36 "pauseAll()" --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

4. If you run 
```
cast call --rpc-url http://localhost:8545/ 0xf5059a5D33d5853360D16C683c16e67980206f36 "paused()"
```
the response should be `0xff...ff`

5. To unpause the contract, run
```
cast send --rpc-url http://localhost:8545 0xf5059a5D33d5853360D16C683c16e67980206f36 "unpause(uint256)" 0 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

6. If you run 
```
cast call --rpc-url http://localhost:8545/ 0xf5059a5D33d5853360D16C683c16e67980206f36 "paused()"
```
the response should be `0x0`

# How to Deploy

## New Deployments
Just deploy aligned contracts as always
